### PR TITLE
Strings cleanup

### PR DESCRIPTION
**Pull request by Neviothr**
**When merged this pull request will:**
- do some clean up in strings component.

### CODE REVIEW

File: addons/strings/fnc_find.sqf [hunk 1]
```diff
@@ -35,31 +35,26 @@ SCRIPT(find);
 
 // ----------------------------------------------------------------------------
 
-params ["_haystack","_needle", ["_initialIndex",0]];
+params ["_haystack","_needle", ["_initialIndex", 0]];
```
Comment (Killswitch00) [inline addons/strings/fnc_find.sqf]: `...stack", "_needle...`

File: addons/strings/fnc_formatNumber.sqf [hunk 2]
```diff
@@ -64,7 +64,7 @@ Author:
 
 SCRIPT(formatNumber);
 
-params ["_number", ["_integerWidth",DEFAULT_INTEGER_WIDTH], ["_decimalPlaces",DEFAULT_DECIMAL_PLACES], ["_separateThousands",DEFAULT_SEPARATE_THOUSANDS]];
+params ["_number", ["_integerWidth", DEFAULT_INTEGER_WIDTH], ["_decimalPlaces", DEFAULT_DECIMAL_PLACES], ["_separateThousands",DEFAULT_SEPARATE_THOUSANDS]];
```
Comment (Killswitch00) [inline addons/strings/fnc_formatNumber.sqf]: `, DEFAULT_SEPARATE...`

File: addons/strings/fnc_substring.sqf [hunk 3]
```diff
@@ -27,10 +27,13 @@ Author:
 SCRIPT(substring);
 
 // ----------------------------------------------------------------------------
-params["_string","_startIndex","_endIndex"];
+params["_string", "_startIndex", "_endIndex"];
```
Comment (Killswitch00) [inline addons/strings/fnc_substring.sqf]: `params ["_str...`

File: addons/strings/fnc_formatNumber.sqf [hunk 4]
```diff
@@ -88,4 +88,4 @@ if (_separateThousands) then { // add localized thousands seperator "1,000"
 
 if (_isNegative) then {_return = "-" + _return;}; // re-add negative sign
 
-_return
+_return
```
Comment (Dystopian) [inline addons/strings/fnc_formatNumber.sqf]: Am I the one who don't like this "last return" deleting?

Comment (jonpas) [reply addons/strings/fnc_formatNumber.sqf]: Not just you, there should be an empty newline.

**Comment by Neviothr** [comment]
What the...

**Comment by Neviothr** [comment]
Git fails to see the newline :confused:
https://i.imgur.com/hmBQY2N.png

**Comment by Dystopian** [comment]
there were more than one file :)

File: addons/strings/fnc_find.sqf [hunk 5]
```diff
@@ -35,31 +35,26 @@ SCRIPT(find);
...
+private _return = -1;
 
-if (typeName _haystack != "STRING") exitWith {
-    -1
-};
-if (typeName _needle != "STRING") exitWith {
-    -1
-};
+if (typeName _haystack != "STRING") exitWith {-1};
+if (typeName _needle != "STRING") exitWith {-1};
```
Comment (commy2) [inline addons/strings/fnc_find.sqf]: Should change it to `_var isEqualType ""` too.

Comment (Neviothr) [reply addons/strings/fnc_find.sqf]: Done. Also, changed the private's and exitWith's positions. There's no point in assigning vars if the script might be terminated in the next line.

File: addons/strings/fnc_find.sqf [hunk 6]
```diff
@@ -35,31 +35,26 @@ SCRIPT(find);
...
 
-if(_initialIndex < 1) then {
-    _ret = _haystack find _needle;
+if (_initialIndex < 1) then {
+    _return = _haystack find _needle;
 } else {
-    if(_initialIndex > (count _haystack) ) exitWith {
-        -1
-    };
+    if (_initialIndex > (count _haystack)) exitWith {-1};
```
Comment (commy2) [inline addons/strings/fnc_find.sqf]: reminder for me to fix this

**Pull request merged**